lib/periodic_jobs.rb

```diff
@@ -6,161 +6,200 @@
+
   # Update SupplementalClaim statuses with their Central Mail status
-  mgr.register('45 0 * * *', 'AppealsApi::CleanUpPii')
+  mgr.register('15 * * * *', 'AppealsApi::SupplementalClaimUploadStatusBatch')
+
   # Remove PII from appeal records after they have been successfully processed by the VA
-  mgr.register('30 * * * *', 'AppealsApi::EvidenceSubmissionBackup')
+  mgr.register('45 0 * * *', 'AppealsApi::CleanUpPii')
+
   # Ensures that appeal evidence received "late" (after the appeal has reached "success") is submitted to Central Mail
-  mgr.register('0 23 * * 1-5', 'AppealsApi::DecisionReviewReportDaily')
+  mgr.register('30 * * * *', 'AppealsApi::EvidenceSubmissionBackup')
+
   # Daily report of appeals submissions
-  mgr.register('0 23 * * 1-5', 'AppealsApi::DailyErrorReport')
+  mgr.register('0 23 * * 1-5', 'AppealsApi::DecisionReviewReportDaily')
+
   # Daily report of appeals errors
-  mgr.register('0 8 * * 1-5', 'AppealsApi::DailyStuckRecordsReport')
+  mgr.register('0 23 * * 1-5', 'AppealsApi::DailyErrorReport')
+
   # Daily report of all stuck appeals submissions
-  mgr.register('0 23 * * 7', 'AppealsApi::DecisionReviewReportWeekly')
+  mgr.register('0 8 * * 1-5', 'AppealsApi::DailyStuckRecordsReport')
+
   # Weekly report of appeals submissions
-  mgr.register('0 5 * * 1', 'AppealsApi::WeeklyErrorReport')
+  mgr.register('0 23 * * 7', 'AppealsApi::DecisionReviewReportWeekly')
+
   # Weekly CSV report of errored appeal submissions
-  mgr.register('0 0 1 * *', 'AppealsApi::MonthlyStatsReport')
+  mgr.register('0 5 * * 1', 'AppealsApi::WeeklyErrorReport')
+
   # Email a decision reviews stats report for the past month to configured recipients first of the month
-  mgr.register('0 2,9,16 * * 1-5', 'AppealsApi::FlipperStatusAlert')
+  mgr.register('0 0 1 * *', 'AppealsApi::MonthlyStatsReport')
+
   # Checks status of Flipper features expected to be enabled and alerts to Slack if any are not enabled
-  mgr.register('0 0 * * *', 'Crm::TopicsDataJob')
+  mgr.register('0 2,9,16 * * 1-5', 'AppealsApi::FlipperStatusAlert')
+
   # Update static data cache
-  mgr.register('0 0 * * *', 'BenefitsIntakeStatusJob')
+  mgr.register('0 0 * * *', 'Crm::TopicsDataJob')
+
   # Update static data cache for form 526
-  mgr.register('15 * * * *', 'Form526DocumentUploadPollingJob')
+  mgr.register('0 0 * * *', 'BenefitsIntakeStatusJob')
+
   # Update Lighthouse526DocumentUpload statuses according to Lighthouse Benefits Documents service tracking
-  mgr.register('0 3 * * *', 'Form526StatusPollingJob')
+  mgr.register('15 * * * *', 'Form526DocumentUploadPollingJob')
+
   # Updates status of FormSubmissions per call to Lighthouse Benefits Intake API
-  mgr.register('0 2 * * 0', 'Form526StatusPollingJob', 'args' => { paranoid: true })
-  # Checks all 'success' type submissions in LH to ensure they haven't changed
+  mgr.register('0 3 * * *', 'Form526StatusPollingJob')
 
-  # mgr.register('0 0 * * *', 'VRE::CreateCh31SubmissionsReportJob')
```

Comment:
🤔 Does this team know this has been commented out for the past year? I see you're deleting it altogether. Should we tell the team? They made changes to the job in January in https://github.com/department-of-veterans-affairs/vets-api/pull/15306 so that makes me think they think it's running 😕 😕

Reply:
The team was aware.

Reply:
Rachal confirmed she had the team's permission to comment that out.